src/sentry/integrations/vercel/integration.py

```diff
@@ -68,14 +69,93 @@ def get_organization_config(self):
                 "type": "project_mapper",
                 "mappedDropdown": {
                     "items": vercel_projects,
-                    "placeholder": "Select a Vercel Project",  # TOOD: add translation
+                    "placeholder": "Select a Vercel Project",  # TODO: add translation
                 },
                 "sentryProjects": sentry_projects,
             }
         ]
 
         return fields
 
+    def update_organization_config(self, data):
+        # data = {"project_mappings": [[sentry_project_id, vercel_project_id]]}
+
+        metadata = self.model.metadata
+        vercel_client = VercelClient(metadata["access_token"], metadata.get("team_id"))
+        config = self.org_integration.config
+        sentry_project_id = data["project_mappings"][-1][
+            0
+        ]  # TODO: update this to work in the case where a project is removed
+        vercel_project_id = data["project_mappings"][-1][1]
+        sentry_project = Project.objects.get(id=sentry_project_id)
+        sentry_project_dsns = ProjectKey.objects.filter(project=sentry_project).all()
+        enabled_dsns = [dsn for dsn in sentry_project_dsns if dsn.is_active]
+        if enabled_dsns:  # in case there are no enabled DSNs
```

Comment:
@ceorourke If there's no DSN, we should throw a failure and error out. Ideally, we would show that as an actionable error message for the user but if that requires front end changes, we can skip that for now.